sdk/storage/inc/blobs/blob_client.hpp

```diff
@@ -301,5 +306,7 @@ namespace Azure { namespace Storage { namespace Blobs {
     }
 
     friend class BlobContainerClient;
+    friend class ::Azure::Storage::Files::DataLake::DirectoryClient;
+    friend class ::Azure::Storage::Files::DataLake::FileClient;
```

Comment:
Do we need `::Azure::Storage::`?

Reply:
Yes, otherwise it is looking for `Azure::Storage::Blobs::Azure::Storage::Files::DataLake::DirectoryClient`.

Reply:
I doubt that. Can you double-check?

Reply:
In addition, I guess the prefix `::` is not necessary.

Reply:
Well, you are actually right. Removed `::Azure::Storage::`